src/PowerShellEditorServices/Services/DebugAdapter/Debugging/InvalidPowerShellExpressionException.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.PowerShell.EditorServices.Services.DebugAdapter
     /// <summary>
     /// Represents the exception that is thrown when an invalid expression is provided to the DebugService's SetVariable method.
     /// </summary>
-    public class InvalidPowerShellExpressionException : Exception
+    internal class InvalidPowerShellExpressionException : Exception
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Make this exception 'public'.](https://app.codacy.com/manual/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4979332)